_studio/shared/src/mfx_static_assert_structs.cpp

```diff
@@ -3612,3 +3612,9 @@
         MSDK_STATIC_ASSERT_STRUCT_OFFSET(mfxExtCam3DLut                     ,Table                         ,32   )
     #endif
 #endif //defined (__MFXCAMERA_H__)
+
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
+        MSDK_STATIC_ASSERT_STRUCT_OFFSET(mfxExtPartialBitstreamParam        ,Header                        ,0  )
```

Comment:
Please also add assert on size of the ```mfxExtPartialBitstreamParam``` struct, these asserts starts from line 50 of this file